db-service/lib/cql-functions.js

```diff
@@ -1,4 +1,8 @@
 const StandardFunctions = {
+
+
+  session_context: x => `session_context('${x.val}')`,
```

Comment:
This does the same as in [#188](https://github.com/cap-js/cds-dbs/pull/188/files#diff-d9ef4ae327f7e5954027eae8acf2866e34a6a365a11e60a3dc5ff535a210c33fR258). To prevent the possibility to do SQL injection it only applies this to known variables.

Reply:
Ok. Yes I've seen that in postgres. Risk of SQL injection might be 0 as no user input ever goes into there, I think.